docs/section5/Rule5-3.md

```diff
@@ -0,0 +1,34 @@
+
+# Envelope - Rule 5-3  
+
+**Rule ID:** 5-3  
+**Rule Description:** Baseline roof assemblies must conform with assemblies detailed in Appendix A.  
+**Appendix G Section:** Section G3.1-5(b) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic:  
+
+- Get surface conditioning category dictionary for B_RMR: ```scc_dictionary_b = get_surface_conditioning_category(B_RMR)```
+
+- For each building segment in the Baseline model: ```for building_segment_b in B_RMR.building.building_segments:```  
+
+  - For each thermal_block in building segment: ```for thermal_block_b in building_segment_b.thermal_blocks:```  
+
+    - For each zone in thermal block: ```for zone_b in thermal_block_b.zones:```  
+
+      - For each surface in zone, get surface type, ```for surface_b in zone_b.surfaces:```  
+
+        - If surface is roof or ceiling and is regulated, get surface construction: ```if ( ( get_opaque_surface_type(surface_b) == "ROOF" ) AND ( scc_dictionary_b[surface_b.id] != UNREGULATED ) ): construction_b = surface_b.construction```
+
+          **Rule Assertion:**  
+
+          Case 1: Surface construction is specified with layers and a U-factor or R-value is provided: ```if (  ( construction_b.surface_construction_input_option == "LAYERS" ) AND ( construction_b.u_factor OR construction_b.r_value ) ): PASS```  
+
```

Comment:
**AND ( construction_b.u_factor OR construction_b.r_value ) )** what does this condition mean? Is this just a check that a value is provided? If so, I don't think r_value should be included as I believe it's optional.

Reply:
The intention is if "either U-factor or R-value" is provided. Is U-factor required and R-value is optional? Or either one of them is required and the other one is optional?

Reply:
I think U-factor must be provided and r-value is irrelevant. r-value is described as "r-value of the insulation for the surface". I am not really sure what it means. For example, a wall may have brick siding + continuous insulation + steel framing w/cavity insulation + gypsum board in interior. U-factor would include all of these and account for thermal bridging through steel framing. What would be written to r-value field? I suggest leaving it out of the RDS. @JasonGlazer and @supriyagoel - do you have different thoughts?

Reply:
Tables G3.4-x don't even provide R-values, just U-values

Reply:
@JasonGlazer so do you agree that r_value should be ignored by the RCT and the RDSs?

Reply:
No r-value for the baseline. It is not as clear on the proposed side.

Reply:
R_value is removed. And rename this rule from 5-3 to 5-4 as per TCD v8. @mkarpman @JasonGlazer @supriyagoel